src/app/routes/enrolment/models/enrolment-claim.ts

```diff
@@ -15,6 +16,8 @@ export class EnrolmentClaim
   organization: string;
   application?: string;
   requestDate: Date;
+  expirationStatus: string;
+  expirationDate: Date;
```

Comment:
```suggestion
  expirationDate?: Date;
```
This value might not be set.